CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -586,7 +586,7 @@ public static bool TryParseUnit([NotNullWhen(true)]string? str, out {_unitEnumNa
         /// <param name=""provider"">Format to use when parsing number and unit. Defaults to <see cref=""CultureInfo.CurrentCulture"" /> if null.</param>
         public static bool TryParseUnit([NotNullWhen(true)]string? str, IFormatProvider? provider, out {_unitEnumName} unit)
         {{
-            return UnitsNetSetup.Default.UnitParser.TryParse<{_unitEnumName}>(str, provider, out unit);
+            return UnitParser.Default.TryParse(str, Info, provider, out unit);
```

Comment:
Nitpick, but let's consistently use either Info or Info.UnitInfos overload. The latter saves an indirection.

Reply:
I thought the same thing as I was reviewing this today, but when I tried to change it- i realized that unlike the `ParseUnit` version, this would require two statements:

`if(UnitParser.Default.TryParse(.., out unitInfo)){..}`

and every line adds another `10K` or so to the file size, so I suggest we leave it as an `file-size-reduction` optimization.

Reply:
This can probably go into an extension method too at some point, right?

Reply:
I haven't got the preview installed- but from what I gather yes, that should work as a static extension.